src/dcat_ap_no_validator_service/adapter/ontology_graph_adapter.py

```diff
@@ -8,12 +8,22 @@
         "1": {
             "id": "1",
             "name": "The ontologies used by DCAT-AP-NO",
-            "description": "This document specifies the ontology information needed by the DCAT-AP-NO validator tool.",  # noqa
+            "description": "This document specifies the ontology information needed for DCAT-AP-NO by the validator tool.",  # noqa
             "version": "0.1",
             "url": "https://raw.githubusercontent.com/Informasjonsforvaltning/dcat-ap-no/develop/shacl/ontologies.ttl",  # noqa
             "specification_name": "DCAT-AP-NO",
             "specification_version": "2.0",
-            "specification_url": "https://data.norge.no/specification/dcat-ap-no/v2",  # noqa
+            "specification_url": "https://data.norge.no/specification/dcat-ap-no/v2",
+        },
+        "2": {
+            "id": "2",
+            "name": "The ontologies used by SKOS-AP-NO-Begrep",
+            "description": "This document specifies the ontology information needed for SKOS-AP-NO-Begrep by the validator tool.",  # noqa
+            "version": "0.1",
+            "url": "https://raw.githubusercontent.com/Informasjonsforvaltning/skos-ap-no-begrep/develop/ontology/skosno-v1.ttl",  # noqa
```

Comment:
Antagelegvis er det fleire ontologiar som trengs for å validere begrepskataloger. Det kan vere lurt å lage ei ontologies.ttl og importere disse inn til den.

Reply:
Ok, kan jo det. Hvilke ontologier skal jeg legge til?

Reply:
Det må testes ut. Eg får i alle fall 18 feil når eg forsøker å validere med følgande input:
data: https://raw.githubusercontent.com/Informasjonsforvaltning/skos-ap-no-begrep/develop/ontology/skosno-v1.ttl
regelsett: https://raw.githubusercontent.com/Informasjonsforvaltning/skos-ap-no-begrep/develop/shacl/SKOS-AP-NO-Begrep-shape_shape_v1.ttl
ontologi: https://raw.githubusercontent.com/Informasjonsforvaltning/skos-ap-no-begrep/develop/ontology/skosno-v1.ttl
Hypotsen er at enkelte av disse feilene er fordi validatoren ikkje har dei ontologiane som trengs.

Ein automatisk test kan også settast opp for å kaste lys over dette.

Reply:
Du validerer en ontologi?

Reply:
> Du validerer en ontologi?

Nei, det var ein typo i min kommentar. Korrekt lenke for data er:
https://concept-catalog.fellesdatakatalog.digdir.no/collections/960478150